src/modules/AdminModule.cpp

```diff
@@ -811,7 +816,12 @@ void AdminModule::handleGetChannel(const meshtastic_MeshPacket &req, uint32_t ch
 void AdminModule::reboot(int32_t seconds)
 {
     LOG_INFO("Rebooting in %d seconds\n", seconds);
-    screen->startRebootScreen();
+    screen->startAlert([](OLEDDisplay *display, OLEDDisplayUiState *state, int16_t x, int16_t y) -> void {
+        uint16_t x_offset = display->width() / 2;
+        display->setTextAlignment(TEXT_ALIGN_CENTER);
+        display->setFont(FONT_MEDIUM);
+        display->drawString(x_offset + x, 26 + y, "Rebooting...");
+    });
```

Comment:
It feels like now we're repeating this same piece of code whenever we want to reboot / shutdown, rather than calling it from a central location.

Reply:
Made a helper function that just takes the string, to get rid of this boilerplate.